expirationd.lua

```diff
@@ -45,7 +45,11 @@ local constants = {
     -- factor for recalculation of vinyl space size
     default_vinyl_assumed_space_len_factor = 2,
     -- default function on full scan
-    default_on_full_scan = function(...) end
+    default_on_full_scan = function(...) end,
+    -- default function for start_key
+    start_key = function(...) return nil end,
```

Comment:
AFAIK `...` can't be jitted. I think we could stay arguments empty

Reply:
Fixed for all code expirationd.lua